wandb/apis/public/artifacts.py

```diff
@@ -56,7 +56,7 @@
 }"""
 
 
-class ProjectArtifactTypes(Paginator):
+class ArtifactTypes(Paginator):
     QUERY = gql(
         """
         query ProjectArtifacts(
```

Comment:
nit: should this be `query ProjectArtifactTypes`?

Reply:
Yeah, but changing the query name would require changing it in `mock_server.py` which is annoying, so I left it as is. Query names don't matter much.